guacamole/src/main/webapp/translations/ko.json

```
@@ -589,51 +589,51 @@
 
         "FIELD_HEADER_BACKSPACE"    : "<Backspace> 키가 보내는 문자:",
         "FIELD_HEADER_COLOR_SCHEME" : "색상 구성표:",
-        "FIELD_HEADER_COMMAND"      : "Execute command:",
-        "FIELD_HEADER_CREATE_RECORDING_PATH" : "Automatically create recording path:",
-        "FIELD_HEADER_CREATE_TYPESCRIPT_PATH" : "Automatically create typescript path:",
-        "FIELD_HEADER_DISABLE_COPY"  : "Disable copying from terminal:",
-        "FIELD_HEADER_DISABLE_PASTE" : "Disable pasting from client:",
-        "FIELD_HEADER_FONT_NAME"     : "Font name:",
-        "FIELD_HEADER_FONT_SIZE"     : "Font size:",
-        "FIELD_HEADER_ENABLE_SFTP"   : "Enable SFTP:",
-        "FIELD_HEADER_HOST_KEY"      : "Public host key (Base64):",
-        "FIELD_HEADER_HOSTNAME"      : "Hostname:",
-        "FIELD_HEADER_LOCALE"        : "Language/Locale ($LANG):",
-        "FIELD_HEADER_USERNAME"      : "Username:",
-        "FIELD_HEADER_PASSWORD"      : "Password:",
-        "FIELD_HEADER_PASSPHRASE"    : "Passphrase:",
-        "FIELD_HEADER_PORT"          : "Port:",
-        "FIELD_HEADER_PRIVATE_KEY"   : "Private key:",
-        "FIELD_HEADER_SCROLLBACK"    : "Maximum scrollback size:",
-        "FIELD_HEADER_READ_ONLY"     : "Read-only:",
-        "FIELD_HEADER_RECORDING_EXCLUDE_MOUSE"  : "Exclude mouse:",
-        "FIELD_HEADER_RECORDING_EXCLUDE_OUTPUT" : "Exclude graphics/streams:",
-        "FIELD_HEADER_RECORDING_INCLUDE_KEYS"   : "Include key events:",
-        "FIELD_HEADER_RECORDING_NAME" : "Recording name:",
-        "FIELD_HEADER_RECORDING_PATH" : "Recording path:",
-        "FIELD_HEADER_SERVER_ALIVE_INTERVAL" : "Server keepalive interval:",
-        "FIELD_HEADER_SFTP_DISABLE_DOWNLOAD" : "Disable file download:",
-        "FIELD_HEADER_SFTP_ROOT_DIRECTORY"   : "File browser root directory:",
-        "FIELD_HEADER_SFTP_DISABLE_UPLOAD"   : "Disable file upload:",
-        "FIELD_HEADER_TERMINAL_TYPE"   : "Terminal type:",
-        "FIELD_HEADER_TIMEZONE"        : "Time zone ($TZ):",
-        "FIELD_HEADER_TYPESCRIPT_NAME" : "Typescript name:",
-        "FIELD_HEADER_TYPESCRIPT_PATH" : "Typescript path:",
-        "FIELD_HEADER_WOL_BROADCAST_ADDR" : "Broadcast address for WoL packet:",
-        "FIELD_HEADER_WOL_MAC_ADDR"       : "MAC address of the remote host:",
-        "FIELD_HEADER_WOL_SEND_PACKET"    : "Send WoL packet:",
-        "FIELD_HEADER_WOL_WAIT_TIME"      : "Host boot wait time:",
+        "FIELD_HEADER_COMMAND"      : "명령 실행:",
+        "FIELD_HEADER_CREATE_RECORDING_PATH" : "레코딩 경로 자동 생성:",
+        "FIELD_HEADER_CREATE_TYPESCRIPT_PATH" : "Typescript 경로 자동 생성:",
+        "FIELD_HEADER_DISABLE_COPY"  : "터미널에서 복사 사용 안 함:",
+        "FIELD_HEADER_DISABLE_PASTE" : "클라이언트에서 붙여넣기 사용 안 함:",
```

```suggestion
        "FIELD_HEADER_DISABLE_COPY"  : "터미널에서 복사 비활성화:",
        "FIELD_HEADER_DISABLE_PASTE" : "클라이언트에서 붙여넣기 비활성화:",
```
$616, $618과 통일성을 위해 수정 제안합니다!

---

수정완료했습니다 좋은 제안 감사합니다!